lib/jira/commands/tickets.rb

```diff
@@ -36,9 +39,9 @@ def rows
         metadata['issues'].map do |issue|
           [
             issue['key'],
-            issue['fields']['assignee']['name'] || 'Unassigned',
-            issue['fields']['status']['name'] || 'Unknown',
-            truncate(issue['fields']['summary'], 45)
+            (issue['fields']['assignee']['name'] unless issue['fields']['assignee'].nil?) || 'Unassigned',
```

Comment:
hmm, what the error? `fetch` seems to have worked in your other pull
ill probably end up just merging this and testing it myself